docs/prebuild.js

```diff
@@ -79,8 +79,32 @@ async function buildAnalyzerRulesData() {
   await writeFile(targetFile, JSON.stringify(rules, undefined,  '  '));
 }
 
+async function buildSamples() {
+  const targetDir = join(generatedDataDir, 'samples');
+  await mkdir(targetDir);
+
+  const sampleProject = '../samples/Riok.Mapperly.Sample';
+  const projectFilesToCopy = ['CarMapper.cs', 'Car.cs', 'CarDto.cs'];
+  const generatedMapperFile = join(sampleProject, 'obj/Debug/net7.0/generated/Riok.Mapperly/Riok.Mapperly.MapperGenerator/CarMapper.g.cs');
+
+  // clean target directory
+  await deleteFilesWithExtension(targetDir, '.cs');
+
+  // build sample project
+  await execPromise(`dotnet build ${sampleProject}`);
```

Comment:
The project should always be built at this time (same precondition for the api docs).

Reply:
Removed